### DISCUSSION TITLE
Compiling v13.0.0 Qasim Tasmota32 get error sdkconfig.h: No such file or directory

### DISCUSSION THREAD

**Discussion by Bebuccio** [arendst/Tasmota#19038] [Q&A] [answered]
I am trying to compile a customized versione of tasmota 32 using gitpod (https://gitpod.io/#https://github.com/arendst/Tasmota/tree/master).
Without change anything in the developer desk (leaving untouched every files), if I try to compile tasmota32 using the command "platformio run -e tasmota32" I get several error like this:
cc1: fatal error: /home/gitpod/.platformio/platforms/espressif32/builder/sdkconfig.h: No such file or directory

What is the problem?
Is the 13.0.0 version not yet usable in the gitpodio development environment?

**Comment by barbudor** [comment]
I confirm both for master and development branches

@Jason2866 

![image](https://github.com/arendst/Tasmota/assets/2996042/1a52f2a8-0634-4ca8-baca-74f676f3913d)

**Comment by Jason2866** [comment]
Can reproduce. Strange one. Currently no idea why it fails.

**Comment by barbudor** [comment]
@Bebuccio We are investigating but are far from a solution
Even my local VSCode setup is now broken

**Comment by Noschvie** [comment]
TasmoCompiler v10-development can be used as workaround, compiled tasmota32 successfully.

**Reply by Jason2866** [reply]
No, it fails too. Just tested.

**Comment by Jason2866** [comment]
It is a Platformio bug. Issue https://github.com/platformio/platformio-core/issues/4683
Pio Core was updated today.

fyi @barbudor

**Reply by barbudor** [reply]
so the problem is in 6.1.8 ?
I actually saw platformio upgrading itself when starting VSCode
Does it means that docker-tasmota and CI will also fail starting today until this is fixed (or you get a way to force roll back to 6.1.7 ?)

**Comment by Jason2866** [comment]
Yes, everything will fail when pio core is upgraded to 6.1.8

**Comment by Jason2866** [comment]
Quick solution to downgrade from bash:
`~/.platformio/penv/bin/pip3 install --force-reinstall -v platformio==6.1.7`

**Comment by Jason2866** [comment, marked as answer]
Final solution :-)
Use the inbuilt function of VSC Platformio to upgrade and use the just released version 6.1.9
https://github.com/platformio/platformio-core/releases/tag/v6.1.9

Big Thx to @ivankravets from Platformio for this fast fix!!  You and your crew do a great Job :-)
